app/main.js

```diff
@@ -6,11 +6,13 @@ const logger = require('./logger');
 const getAirportsByCountry = require('./getAirportsByCountry');
 const getAirport = require('./getAirport');
 const getCharts = require('./getCharts');
+const { count } = require('console');
 
 commander.program
   .option('--icao <ICAO_CODE>', 'Scrape only one specific airport by airport ICAO code')
   .option('--country <ISO_CODES>', 'Scrape all airports in specific countries by country ISO code')
   .option('--all', 'Scrape all airports in all the countries')
+  .option('--skip-country, --skipcountries [value...]', 'Skip these coutries by ISO code')
```

Comment:
'--skip-country <ISO_CODES>', 'Skip scraping for all airports in specific countries by country ISO code'